bgpd/bgp_route.c

```diff
@@ -10440,7 +10440,7 @@ DEFUN (show_ip_bgp,
            |community-list <(1-500)|WORD> [exact-match]\
            |A.B.C.D/M longer-prefixes\
            |X:X::X:X/M longer-prefixes\
-	  >",
+	  > [json]",
```

Comment:
As a note with this change you just added json support for dampening <parameters>, route-map WORD, prefix-list WORD, filter-list WORD.... none of which I see actual json code for?

Reply:
This is the point. I dont plan to do json for dampening, prefix list.  but perhaps someone would like to make support for this. 
from your comment, however, I will do separate vty command for that.